drivers/volume/portworx/portworx.go

```diff
@@ -469,7 +528,12 @@ func (p *portworx) GetNodes() ([]*storkvolume.NodeInfo, error) {
 		}
 	}
 
-	cluster, err := p.clusterManager.Enumerate()
+	clusterManager, err:= p.getClusterManagerClient()
```

Comment:
Why get cluster manager again?

Why was it removed from the object `p`?

Reply:
When cluster manager had static config we could just keep it as part of `p *portworx` struct. Now when we need to populate cluster manager client with token which has relatively short lifetime (1 hour), `getClusterManagerClient()` is used as getter which returns fully configured client with fresh token injected.